ArduPlane/quadplane.cpp

```diff
@@ -2228,16 +2310,16 @@ void QuadPlane::vtol_position_controller(void)
     }
 
     case QPOS_LAND_DESCEND: {
-        float height_above_ground = plane.relative_ground_altitude(plane.g.rangefinder_landing);
-        pos_control->set_alt_target_from_climb_rate(-landing_descent_rate_cms(height_above_ground),
-                                                    plane.G_Dt, true);
+        cmb_rate = compute_descent_rate(2.0f, doing_precision_landing);
+        pos_control->set_alt_target_from_climb_rate_ff(cmb_rate, plane.G_Dt, true);
```

Comment:
why the change to use the _ff method?
can't we just change landing_descent_rate_cms() if we do need a different rate for precland? (and do we?)

Reply:
Not sure the motivation behind this change, left it in because it seemed to be implicitly approved by nobody complaining about it in the previous PR it appeared in. It doesn't make sense to me, I'll remove it because it doesn't seem in the scope of the PR anyway